conf.d/main

```diff
@@ -18,13 +18,38 @@ CONF=$WEBROOT/.env
 LOC_SHARE=/usr/local/share
 LOC_BIN=/usr/local/bin
 
+#install inoncube loader
+systemctl restart apache2
```

Comment:
I suggest doing this before you start apache, so then this line is redundant.

Reply:
Would the code for installation still go in the same file?